knowledge-base/actions/arduino/arduino-lint-action/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: "arduino/arduino-lint-action"
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
```

Comment:
@vandana41 I think there is only `content: read` permission needed to get `git tags` for the repos, view [here](https://github.com/arduino/arduino-lint-action/blob/e26fd2724ebc5d1b5f0c32aa575c9f68fab8ab4f/src/installer.ts#L135), [here](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#git-1)

There is no need for adding permissions for KBs having only content: read permission, because it gets added at workflow level by default.
so please remove all the permissions for this KB.